src/components/Firestore/FirestoreEmulatedApiProvider.test.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  act,
+  waitForElement,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import { firestore } from 'firebase';
+import React, { useState } from 'react';
+import { useFirestoreDocData } from 'reactfire';
+
+import { jsonIshValue } from '../Database/DataViewer/common/view_model';
+import {
+  useEjector,
+  useRootCollections,
+  useSubCollections,
+} from './FirestoreEmulatedApiProvider';
+import { renderWithFirestore } from './testing/FirestoreTestProviders';
+
+test('should get root-collections', async () => {
```

Comment:
Is there a reason to prefer test over it, like we do everywhere else?

Reply:
Good catch, updated to `it` for consistency